### DISCUSSION TITLE
HTML bespoke output: Skip the progressive list item slides

### DISCUSSION THREAD

**Discussion by ymollard** [marp-team/marp#215] [Q&A] [answered]
In the html output by `marp-cli` with bespoke theme, pressing the Next button displays the next item.
Is it possible to make all list items appear at once at build time, or with a key press at presentation time?

As a fix I have switched to the bare template that acts this way (`marp --template=bare`), but bespoke is nice.

**Comment by yhatt** [comment, marked as answer]
Marp recognizes the list with `*` marker as the fragmented list.

```markdown
* Step 1
* Step 2
* Step 3
```

To show list at once, use `-` marker instead.

```markdown
- Step 1
- Step 2
- Step 3
```

See also: https://marpit.marp.app/fragmented-list?id=for-bullet-list

> Advanced: You also can make a fragment group by using `<div data-marpit-fragment>`. It requires to enable HTML tags by `--html` option. https://github.com/marp-team/marpit/issues/195
>
> ```markdown
> <div data-marpit-fragment>
> 
> - Step 1
> - Step 2
> - Step 3
> 
> </div>
> ```

**Reply by ymollard** [reply]
Great, thank you!

**Reply by testsmith-io** [reply]
it doesn't work like that if you export images, right?
